src/PKSim.CLI.Core/RunOptions/ExportRunOptions.cs

```diff
@@ -15,6 +15,12 @@ public class ExportRunOptions : Notifier, IValidatable, IWithOutputFolder
       public IEnumerable<string> Simulations { get; set; }
       public bool RunSimulation { get; set; }
 
+      /// <summary>
+      /// Specifies if all simulations should be exported if the provided list of Simulations is empty.
+      /// Default is true and will be overwritten for special use cases
+      /// </summary>
+      public bool ExportAllSimulationsIfListIsEmpty { get; set; } = true;
```

Comment:
Well this was missing. BEfore, if the list was not provided, we assumed all should be exported..